src/app/lib/utilities/parseAssetData/index.js

```diff
@@ -1,12 +1,26 @@
 import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
 
 const getISOStringDate = date => new Date(date).toISOString();
 
 export const getArticleId = path(['metadata', 'id']);
 
 export const getHeadline = path(['promo', 'headlines', 'seoHeadline']);
 
-export const getSummary = path(['promo', 'summary']);
+export const getSummary = pathOr('', [
```

Comment:
Do we not need two methods here - one for `getOptimoSummary` and another for `getCpsSummary`?

Reply:
My mistake. It seems like this method is only used by Optimo pages: https://github.com/bbc/simorgh/blob/5005a6e43adc86c996cebcbd4c43d25c8d525b83/src/app/pages/ArticlePage/index.jsx#L93